WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4ChartAxisHelper.swift

```diff
@@ -0,0 +1,51 @@
+import Yosemite
+
+/// A helper for the labels for store stats v4 charts UI.
+class StoreStatsV4ChartAxisHelper {
```

Comment:
I'm just going to leave my usual comment... 😛  Do we really want to be able to subclass this? Should it be final?

Reply:
my bad! I certainly forgot the new habit to mark a class as final by default 😬will update!